src/utils/data-management/data-management.service.ts

```diff
@@ -240,7 +241,7 @@ export class DataManagementService {
       await this.connection.synchronize();
       this.addLogMsg(msgs, '.....dropped. Completed successfully.');
     } catch (error) {
-      this.logger.verbose(error.message);
+      this.logger.verbose(error.message, LogContexts.DATA_MGMT);
```

Comment:
Check error messages - I think there are 3 parameters for this.logger.error and the second one is not the context.

Reply:
yes but this is a verbose message